pom.xml

```diff
@@ -493,23 +492,20 @@
         <!-- we need to override just make "mvn versions:plugin-updates-report"
              see no old versions -->
         <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-antrun-plugin</artifactId>
           <version>3.1.0</version>
         </plugin>
         <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-assembly-plugin</artifactId>
           <version>3.7.1</version>
         </plugin>
         <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-dependency-plugin</artifactId>
           <version>3.8.1</version>
         </plugin>
-        <plugin>
-          <artifactId>maven-release-plugin</artifactId>
```

Comment:
idk why its done this might be default now. Need to check recipe.

Reply:
It was added for som reason, please investigate git history to get this reason

Reply:
if you want to apply let me know. As quick rebase with merge will be needed to avoid conflicts.